File: BusinessLogic/PayIn.Application.Dto/Results/ControlIncident/ControlIncidentCreateManualCheckResult.cs
using System.Collections.Generic;
using Xp.Common;

namespace PayIn.Application.Dto.Results.ControlIncident
{
	public class ControlIncidentCreateManualCheckResult
	{
		public class Track
		{
			public int Id { get; set; }
		}

		public IEnumerable<Track> Entrances      { get; set; }
		public IEnumerable<Track> Exits          { get; set; }
		public XpTime             TrackFrecuency { get; set; }
	}
}

File: AndroidApp/PayIn/app/src/main/java/com/mobile/application/payin/dto/results/ControlIncidentCreateManualCheckResult.java
package com.mobile.application.payin.dto.results;

import java.io.Serializable;
import java.util.ArrayList;

public class ControlIncidentCreateManualCheckResult implements Serializable {
    public ArrayList<Track> Entrances;
    public ArrayList<Track> Exits;
    public String           TrackFrecuency;

    public ControlIncidentCreateManualCheckResult() {
        Entrances = new ArrayList<>();
        Exits = new ArrayList<>();
    }

    public static class Track implements Serializable {
        public int Id;

        public Track() {}
    }
}
